src/functions-and-arrays.js

```diff
@@ -1,38 +1,91 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
-
+function maxOfTwoNumbers(n1, n2) {
+  if (n1 > n2) {
+      return n1;
+    } else {
+      return n2;
+  }
+}
 
 
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
-function findLongestWord() {}
+function findLongestWord(words) {
+  if (words.length === 0) {
+    return null;
+  }
+  let longest = words[0];
+  for (let i = 0; i < words.length; i++) {
```

Comment:
```suggestion
  for (let i = 1; i < words.length; i++) {
```

Si te fijas la primera posicion ya la estas cogiendo antes, asi que compararla consigo misma pues como que no tiene mucho sentido jejeje